### ISSUE TITLE
Add randomness to hot-starting in GaussianFusionHMM

### ISSUE THREAD

**Issue by rmcgibbo** [msmbuilder/msmbuilder#66]
In any more on an input space with more than a small handful of degrees of freedom, there are going to be numerous local minima in the hmm likelihood function. The way to be honest about this, IMO, is to make sure that our hot-start is random enough that running the code multiple times with different random seeds will fall into different local minima. Currently, `GaussianFusionHMM` has a `random_state` argument to `__init__`, but it's ignored.
- One output would be to make the [KMeans](http://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html) [hot-starting](https://github.com/rmcgibbo/mixtape/blob/master/Mixtape/ghmm.py#L224) use random initialization instead of KMeans++, with the random seed passed through.
- We could also add an option to the command line to run multiple repetitions with different random seeds.

cc: @gkiss, @rbharath
